### DISCUSSION TITLE
v24.2.16: API - Status Response - important reporting variables are missing

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#1760] [Ideas]
Hello,
I migrated to the new version today and discovered that the API output has changed significantly.
I think it's very good that they're now also offering smaller responses.

I find the current implementation very critical, as essential information for the control (e.g. zero feed-in) is missing.
I have currently gone back to the old version.

I would like it as follows:

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/7007abb2-f969-46ce-8cc0-1bbf40fb9ce8)

Otherwise I have to query the inverters one after the other. That can't be the goal.
Thanks for the check! (If I have missed something, please let me know!)

note:
https://github.com/tbnobody/OpenDTU/discussions/1463
https://github.com/tbnobody/OpenDTU/issues/1615#issuecomment-1902468777

**Comment by 2000ede** [comment]
@MetaChuh
can you please look at this? Thanks!

**Comment by MetaChuh** [comment]
@2000ede 

> @MetaChuh
can you please look at this? Thanks!

we did look at it,
but at least i think do not fully understand what can be missing,
as all listed values from above are available via the opendtu api, albeit not from a single api query.

**Reply by 2000ede** [reply]
@MetaChuh 

You rightly want to relieve the load on OpenDTU.

However, the basic API call should contain the minimum sizes for zero feed. This is currently not the case. **The blue values ​​are missing.**
If you don't know what the individual inverter produces, it's difficult to set a good limit.

Unfortunately, this will mean that you will have to make/have to make the other large calls again in order to achieve zero feed-in.
On the contrary, it has actually become more complicated for the user.
This is unnecessary and counterproductive.

**Comment by 2000ede** [comment]
That's not good. And so I don't dare to express another wish. Unfortunately, I always have to count the digits with Tasmota to get the right number. If the number of inverters or modules changes, I unfortunately have to adjust it again.

It would be nice if the values ​​in the response were always clear, such as "limit_absolute".

(sorry, my english is bad)

**Comment by MetaChuh** [comment]
> That's not good. And so I don't dare to express another wish.

please don't hesitate to ask any questions or wishes.

> (sorry, my english is bad)

if you like, you can add your questions in your native language to your prior requests below your original question, by using the github `edit` function at the `...` menu of each post.

as an international community, there's a high probability, that one of us shares the same native language, and has similar experiences and/or solutions.

greetings,
metachuh
